tests/_support/Page/ENBlock.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Page;
+
+class ENBlock {
+	/**
+	 * Declare UI map for this page here. CSS or XPath allowed.
+	 * public static $usernameField = '#username';
+	 * public static $formSubmitButton = "#mainForm input[type=submit]";
+	 */
+
+	// include url of current page
+	public static $URL = '/wp-admin/post-new.php?post_type=p4en_form';
```

Comment:
Could you please align the equal signs here to make it look better.
Also, can we make these constants instead of static since I guess they are not changing, right?

Reply:
Yeap they are not changing.
It is meant to implement the PageObject pattern.
This is how the default codeception generator creates the file.
https://codeception.com/docs/06-ReusingTestCode#PageObjects

It does not make any huge difference, but it looks a bit prettier than constants I think.
What do you think?

```
	const GOALSELECT = 'enform_goal';

		$I->selectOption( ENBlock::GOALSELECT, $option );
```

```
	public static $goalSelect = 'enform_goal';

		$I->selectOption( ENBlock::$goalSelect, $option );
```

Reply:
sure, we can let it like this